src/test/config.sol

```diff
@@ -95,9 +95,9 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x230BFbDbBbA8F07282A67ed4868aC261F23272F2),     // populate with deployed spell if deployed
-            deployed_spell_created:         1676905116,              // use `./scripts/get-created-timestamp.sh <deployment-tx>`
-            deployed_spell_block:           8525393,        // populate with the block where the spell was deployed
+            deployed_spell:                 address(0x5Cc537ed3C9A286128f14376553476dCD0F3C5AB),     // populate with deployed spell if deployed
+            deployed_spell_created:         1677847836,              // use `./scripts/get-created-timestamp.sh <deployment-tx>`
```

Comment:
btw the comment here seems to be outdated, looks like now we use `scripts/time.sh` with the formatted timestamp as the argument

Reply:
Comment was outdated but it is not that script either. I have fixed it.